docs/content/Installation/postgres-operator-installer/_index.md

```diff
@@ -10,7 +10,7 @@ weight: 100
 The following jobs can be run to install, update, and uninstall the Crunchy
 PostgreSQL Operator in your Kubernetes or OpenShift cluster using the
 pgo-installer image. Examples of these job can be found in
-`$PGOROOT/installers/pgo-installer/ansible/roles/pgo-installer/templates`. The
+`$PGOROOT/installers/method/playbook/roles/pgo-installer/templates`. The
```

Comment:
`method` seems odd -- especially given below it shows `installers/ansible` -- I'm not sure I understand this choice?

Reply:
The idea behind the `method` directory is that it can be a home for other installation methods that can be used as a wrapper for the `pgo-installer` image. Since `installers/ansible` and the other installers in `installers` don't act as wrappers for the image (yet) i didn't move them.  For example once olm or helm are updated to use the installer image they would be added to method.

Reply:
Perhaps we call it `ansible` instead of `playbook` then? or `ansible-playbook`? Just seems a bit confusing.

I agree with the thought process overall, though :smile:

Reply:
Thats fair. I assumed people would know what `playbook` meant. I tried to stay away from `ansible` to limit confusion with the other `ansible`. I think `ansible-playbook` would work so ill go with that

Reply:
:+1: